.gitignore

```diff
@@ -2,8 +2,5 @@ _site
 .sass-cache
 .jekyll-metadata
 node_modules
-assets/js
-assets/uswds
-_sass/uswds
```

Comment:
I don't know why `assets/js` was in this file given that it is where user js is supposed to go. `assets/uswds` and `_sass/uswds` are no longer used.